src/spec/tap_formatter.cr

```diff
@@ -0,0 +1,30 @@
+# :nodoc:
+class Spec::TAPFormatter < Spec::Formatter
+  @counter = 0
+
+  def report(result)
+    case result.kind
+    when :success
```

Comment:
This should be `when :success, :pending`.

Reply:
Nope, that would reduce visibility. Separate entries makes it obvious what we're looking at.